blog/models.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from blog_system import settings
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="posts"
+    )
+    title = models.CharField(max_length=255)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        ordering = ["-created_time"]
+
+    def __str__(self) -> str:
+        return f"{self.owner}: {self.title} ({self.created_time})"
+
+
+class Commentary(models.Model):
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="commentaries"
+    )
+    post = models.ForeignKey(
+        "Post",
+        on_delete=models.CASCADE,
+        related_name="commentaries"
+    )
+    created_time = models.DateTimeField(auto_now_add=True)
+    content = models.TextField()
+
+    class Meta:
+        ordering = ["-created_time"]
+        verbose_name_plural = "Commentaries"
+
+    def __str__(self) -> str:
+        return f"{self.user}: {self.post} ({self.created_time})"
+
+
+class User(AbstractUser):
+    pass
```

Comment:
why you have pass here, if you have overrode methods inside it?)

Reply:
> why you have pass here, if you have overrode methods inside it?)

forgot to delete)